template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hbase/HbaseCloudStorageServiceConfigProvider.java

```diff
@@ -48,9 +55,12 @@ public boolean isConfigurationNeeded(CmTemplateProcessor cmTemplateProcessor, Te
                 .orElse(false);
         String cdhVersion = getCdhVersion(source);
         boolean is722OrNewer = isVersionNewerOrEqualThanLimited(cdhVersion, CLOUDERAMANAGER_VERSION_7_2_2);
+        GeneralClusterConfigs generalClusterConfigs = source.getGeneralClusterConfigs();
+        boolean sdxHbaseCloudStorageEnabled = entitlementService == null ? true :
```

Comment:
I don't think the EntitlementService can be null.

Reply:
In the test CentralCmTemplateUpdaterTest, I debug into the code, and it is null. That is why I put the condition there to let the test pass. I suspect in real end-to-end test, EntitlementService will not be null. 

What is alternative for CentralCmTemplateUpdaterTest to pass without allowing EntitlementService to be null?

Reply:
You need to include the service in the test as well
```
@Mock
    private EntitlementService entitlementService;
```
and it won't be null

Reply:
The code is updated based on your suggestion.